regparser/tree/xml_parser/appendices.py

```diff
@@ -270,6 +278,8 @@ def is_subhead(tag, text):
             elif initial_marker(text) and child.tag in ('P', 'FP', 'HD'):
                 self.paragraph_with_marker(
                     text, tree_utils.get_node_text_tags_preserved(child))
+            elif child.tag in ('P', 'FP') and child.get('SOURCE') == 'FP-DASH':
+                self.paragraph_with_dash(text)
```

Comment:
This makes sense, but I wonder if `FP-DASH` shouldn't be orthogonal to the marker-or-no-marker decision. Should this be a transform on just text (e.g. `text = insert_dashes(text, child)`)?